crypto/.jvm/src/main/scala/org/bitcoins/crypto/BouncyCastleUtil.scala

```diff
@@ -243,7 +168,9 @@ object AdaptorStuff {
       adaptorPoint: ECPublicKey,
       dataToSign: ByteVector): ECAdaptorSignature = {
     // Include dataToSign and adaptor in nonce derivation
-    val hash = CryptoUtil.sha256(dataToSign ++ serializePoint(adaptorPoint))
+    val hash =
+      CryptoContext.cryptoRuntime.sha256(
```

Comment:
This and the diffs below can be reverted right?

Reply:
Why?

Reply:
`CryptoUtil.sha256` is fine no? Isn't it shorthand for `CryptoContext.cryptoRuntime.sha256`?